src/int.rs

```diff
@@ -717,6 +715,104 @@ impl I256 {
         self.overflowing_rem(rhs).0
     }
 
+    /// Calculates the quotient of Euclidean division of self by rhs.
+    /// This computes the integer n such that self = n * rhs + self.rem_euclid(rhs),
+    /// with 0 <= self.rem_euclid(rhs) < rhs.
+    /// In other words, the result is self / rhs rounded to the integer n such that self >= n * rhs.
+    /// If self > 0, this is equal to round towards zero (the default in Rust);
+    /// if self < 0, this is equal to round towards +/- infinity.
```

Comment:
nit: Doc comments in Rust are markdown, so this will be formatted like so:

> Calculates the quotient of Euclidean division of self by rhs. This computes the integer n such that self = n * rhs + self.rem_euclid(rhs), with 0 <= self.rem_euclid(rhs) < rhs. In other words, the result is self / rhs rounded to the integer n such that self >= n * rhs. If self > 0, this is equal to round towards zero (the default in Rust); if self < 0, this is equal to round towards +/- infinity.

```suggestion
    /// Calculates the quotient of Euclidean division of self by rhs.
    ///
    /// This computes the integer `n` such that `self = n * rhs + self.rem_euclid(rhs)`,
    /// with `0 <= self.rem_euclid(rhs) < rhs`.
    /// In other words, the result is `self / rhs` rounded to the integer `n` such that `self >= n * rhs`:
    /// * If `self > 0`, this is equal to round towards zero (the default in Rust);
    /// * If `self < 0`, this is equal to round towards +/- infinity.
```

Will have nicer formatting:
> Calculates the quotient of Euclidean division of self by rhs.
>
> This computes the integer `n` such that `self = n * rhs + self.rem_euclid(rhs)`,
> with `0 <= self.rem_euclid(rhs) < rhs`.
> In other words, the result is `self / rhs` rounded to the integer `n` such that `self >= n * rhs`:
> * If `self > 0`, this is equal to round towards zero (the default in Rust);
> * If `self < 0`, this is equal to round towards +/- infinity.

Same for other comments.